.github/workflows/docs.yml

```diff
@@ -22,13 +22,21 @@ jobs:
           uses: actions/checkout@v2
           with:
             fetch-depth: 0
+            submodules: recursive
         - name: Set system to non-interactive mode
           run: export DEBIAN_FRONTEND=noninteractive
         - name: install dependencies
           run: |
+            sudo apt-get install -y --force-yes -qq build-essential python3-dev
```

Comment:
In docker, sudo is standard. It's considered safe because you're in a container. We could instead install all dependencies via, e.g., spack, but this is a lot faster and for github I'd prefer to be fast.